release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -47,6 +60,13 @@ class CreateReleaseSubstepsState extends State<CreateReleaseSubsteps> {
     });
   }
 
+  // When isEachInputValid[0] is true, the first parameter is valid. If it false, it is unvalid.
```

Comment:
nit, in English the word is "invalid"
```suggestion
  // When isEachInputValid[0] is true, the first parameter is valid. If it false, it is invalid.
```